test/powershell/engine/BigIntegerCultureHandling.test.ps1

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'PowerShell Type Conversion - BigInteger Parsing' -Tag 'CI' {
+
+    It 'Can convert formatted numbers using PowerShell type system' {
+        [System.Globalization.CultureInfo]::CurrentCulture = [System.Globalization.CultureInfo]::GetCultureInfo("en-US")
+        $formattedNumber = "1,000"
+        $convertedValue = [System.Management.Automation.LanguagePrimitives]::ConvertTo($formattedNumber, [bigint])
+        $convertedValue | Should -Be 1000
+    }
+
+    It 'Handles large comma-separated numbers that previously failed' {
```

Comment:
Quibble: perhaps better: `Handles large numbers with thousands separators that previously failed'